images/win/scripts/Tests/Docker.Tests.ps1

```diff
@@ -1,25 +1,35 @@
 Describe "Docker" {
     It "<ToolName>" -TestCases @(
-        @{ ToolName = "docker" }
         @{ ToolName = "docker-compose" }
     ) {
         "$ToolName --version" | Should -ReturnZeroExitCode
     }
```

Comment:
Do you want to get rid of test cases list as only one element left?

Reply:
no